fix(docs): Information about the plugin app-visualizer made more prominent (#34537)
backstage/backstage: docs/frontend-system/building-apps/09-troubleshooting-app-visualizer.md

---
id: troubleshooting-visualizer
title: Install Troubleshooting Plugin Visualizer
sidebar_label: Troubleshooting Plugin Visualizer
description: How to install the plugin visualizer in the frontend system for troubleshooting
---

## Overview

This section describes how to install the `app-visualizer` plugin in the frontend system. It also describes which views are available.

### Visualizer Plugin

The `app-visualizer` plugin can help with troubleshooting. It provides a visual overview of your app's extension tree, making it easy to verify that plugins are installed correctly, see how extensions are wired together, and identify issues during migration.

> It can only be installed starting with release 1.49.0.

Reviewer: This should be a note, otherwise people may miss it. You may want to consider info on how to figure out what release you are running.

Author: Since we are far beyond the release 1.49.0, line 16 is deleted.

> It can only be installed starting with release 1.49.0.

Author: I have corrected according to this comment.

---

> id: troubleshooting-visualizer

Reviewer 2: ```suggestion
id: app-visualizer
```

This feels like a better fit

> title: Install App Visualizer Plugin

Reviewer 2: ```suggestion
title: App Visualizer
```

There's more to this page than just installation details.

> sidebar_label: App Visualizer Plugin for troubleshooting

Reviewer 2: ```suggestion
sidebar_label: App Visualizer 
```

It's best to keep these short so the sidebar doesn't become very wide.

Author: ok

> description: How to install the  app-visualizer plugin in the frontend system for troubleshooting

Reviewer 2: ```suggestion
description: How to install and use the  App Visualizer plugin in the frontend system for feature and extension discovery.
```
